alignak/objects/contact.py

```diff
@@ -207,36 +211,33 @@ def serialize(self):
 
         return res
 
-    def get_name(self):
-        """Get contact name
+    def get_groupname(self, groups):
+        """Get name of the contact's first contactgroup
 
-        :return: contact name
+        :return: the first contact group name
         :rtype: str
         """
-        try:
-            return self.contact_name
-        except AttributeError:
-            return 'UnnamedContact'
-
-    def get_groupname(self):
-        """
-        Get the first group name whose contact belongs to
-        :return: group name
+        groupname = ''
```

Comment:
Because the macro using this function must return the group alias if it exists. It it does not, the default initialization will make the alias equal to the group name.